libs/utils/android/screen.py

```diff
@@ -136,4 +137,20 @@ def get_screen_density(target):
         """
         return target.execute('getprop ro.sf.lcd_density')
 
+    @staticmethod
+    def set_screen(target, on=True):
+        log = logging.getLogger('Screen')
+        if on == True:
+            target.execute('input keyevent KEYCODE_WAKE')
```

Comment:
I thought you preferred to send the event directly from Screen, I can certainly add a different path with that for System,... actually that's what I was going to do too.